variables.tf

```diff
@@ -0,0 +1,115 @@
+variable "vpc_id" {
+  description = "The ID of the VPC where resources will be deployed"
+  type        = string
+}
+
+variable "corelight_sensor_ami_id" {
+  description = "The AMI ID of the Corelight sensor. Request access to the AMI from your account executive"
+  type        = string
+}
+
+variable "monitoring_subnet_id" {
+  description = "The ID of the subnet where monitor traffic will be available"
+  type        = string
+}
+
+variable "management_subnet_id" {
+  description = "The ID of the subnet used to SSH / manage Corelight sensors"
+  type        = string
+}
+
+variable "aws_key_pair_name" {
+  description = "The name of the AWS key pair that will be used to access the sensor instances in the auto-scale group"
+  type        = string
+}
+
+variable "auto_scaling_availability_zones" {
+  description = "The availability zones the auto scale group will use"
+  type        = list(string)
+}
+
+variable "sensor_api_password" {
+  description = "The password that should be used for the Corelight sensor API"
+  sensitive   = true
+  type        = string
+}
```

Comment:
Can we update to the following:

```
variable "community_string" {
  type        = string
  sensitive   = true
  description = "the community string (api string) often times referenced by Fleet"
}
```

Matches with https://github.com/corelight/terraform-gcp-sensor/pull/2